src/styles/blocks/button.scss

```diff
@@ -0,0 +1,37 @@
+.button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  width: 100%;
+  height: 56px;
+  border: 2px solid $blue-color;
+  border-radius: 8px;
+
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 14px;
+  color: $blue-color;
+  transition: all $transition-duration;
```

Comment:
I recommend not to use transition all, but to indicate which properties you want to use it for